microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md

```diff
@@ -151,6 +151,19 @@ See also:
 * [Manifest file format for extensions](../getting-started/manifest-format.md)
 
 
+<!-- ====================================================================== -->
+## Blocking of extensions
```

Comment:
C.D. -- is this section heading good?  Should this section be added to this article (which user filed Issue on)?

issue #2436 (Interference of BlockExternalExtensions is unclear)

Reply:
Mike, This looks good to me.

Reply:
i will resolve this thread after i delete the hidden comment that shows the orig draft